src/framework/preferences/css/stylus/Enactors.styl

```diff
@@ -25,9 +25,11 @@
     font-family: "Comic Sans MS", sans-serif !important;
 }
 
-// UI Enhancer "Enlarge text" link options
+// UI Enhancer "Enlarge text" input/button options
 .fl-text-larger input,
-.fl-text-larger button {
+.fl-text-larger button,
+.fl-text-larger .fl-input-enhance,
+.fl-text-larger .fl-button-enhance, {
```

Comment:
@cindyli those are both new class names. In previous versions of UIO there was no support for enhancing custom inputs and buttons. These are provided for integrators and us to use for those cases. Alternatively an integrator could just worry about it themselves, but I figured if we're adding one for inputs, we could do the same for buttons. Or we could just ask them to drop on the `fl-input-enhance` onto their buttons.